db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/DecimalBufferedSumAggregator.java

```diff
@@ -61,6 +61,11 @@ public DecimalBufferedSumAggregator() { // SERDE
 
 	}
 
+	public DecimalBufferedSumAggregator(CatalogMessage.SystemAggregator agg) throws IOException, ClassNotFoundException {
+		this(64);
```

Comment:
why 64?

Reply:
I'm copying the size from else where. Not sure why it was decided to be 64

Reply:
Jun, can you research this one offline?  No need to hold the submit, but would be nice to have an explanation for the 64.

Reply:
Please make this part of the serialized message